syntax = "proto3";

package user;

service UserService {
    rpc CreateUser(UserRequest) returns (UserResponse);
}

message UserRequest {
    string name = 1;
    string phone = 2;
    string email = 3;
}

message UserResponse {
    string message = 1;
}